cmd/main.go

```diff
@@ -100,10 +100,9 @@ func buildCmdTree() *cobra.Command {
 	createCmd := newCreateCmd()
 	// kubebuilder create api
 	createCmd.AddCommand(newAPICmd())
-	// kubebuilder create webhook (v2 only)
-	if !internal.ConfiguredAndV1() {
-		createCmd.AddCommand(newWebhookV2Cmd())
-	}
+	// kubebuilder create webhook
+	createCmd.AddCommand(newWebhookV2Cmd())
```

Comment:
Probably rename `newWebhookV2Cmd` to `newWebhookCmd`

Reply:
done